app/lib/publisher/backend.dart

```diff
@@ -217,6 +217,20 @@ class PublisherBackend {
     });
   }
 
+  /// The list of e-mail addresses of the members with admin roles. The list
+  /// should be used to notify admins on upload events.
+  Future<List<String>> getAdminMemberEmails(String publisherId) async {
+    final result = <String>[];
+    final key = _db.emptyKey.append(Publisher, id: publisherId);
+    final query = _db.query<PublisherMember>(ancestorKey: key)
+      ..filter('role =', PublisherMemberRole.admin);
```

Comment:
Why not just email all members of the publisher... I don't think we need to filter on email...

Reply:
Sure, let's start with that.